src/lightning_app/utilities/packaging/cloud_compute.py

```diff
@@ -116,6 +131,13 @@ def is_default(self) -> bool:
         return self.name == "default"
 
 
+def _verify_mount_root_dirs_are_unique(mounts: Union[None, Mount, List[Mount], Tuple[Mount]]) -> None:
+    if isinstance(mounts, (list, tuple, set)):
```

Comment:
same question here, the type annotation does not match the instance check.

Reply:
see reply https://github.com/Lightning-AI/lightning/pull/15120#discussion_r999393624